business-central-parent/pom.xml

```diff
@@ -155,21 +155,23 @@
           </executions>
         </plugin>
         <plugin>
-          <groupId>io.thorntail</groupId>
-          <artifactId>thorntail-maven-plugin</artifactId>
+          <groupId>org.wildfly.plugins</groupId>
+          <artifactId>wildfly-jar-maven-plugin</artifactId>
           <configuration>
-            <fractionDetectMode>never</fractionDetectMode>
-            <fractions>
-              <fraction>logging</fraction>
-              <fraction>undertow</fraction>
-              <fraction>cdi</fraction>
-              <fraction>ejb</fraction>
-              <fraction>jaxrs</fraction>
-              <fraction>ee</fraction>
-              <fraction>security</fraction>
-              <fraction>management</fraction>
-              <fraction>datasources</fraction>
-            </fractions>
+<!--            <feature-pack-location>org.jboss.eap:wildfly-galleon-pack:${version.server.standalone-jar}</feature-pack-location>-->
+<!--            <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#${version.server.bom}</feature-pack-location>-->
+            <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#23.0.0.Final</feature-pack-location>
```

Comment:
Wildfly version correctly hard coded in for cli-script handle.

Reply:
just question is that depending on version property declared in kie-parent?

Reply:
Right now not, but that is what needs to be done to make this clean. It is possible to define any version here and not match the main one used for the rest of the projects.

Reply:
ok

Reply:
i think this can be now changed to version property from kie-parent pom.xml